#!/Users/surajshashidhar/anaconda3/envs/django_environment1/bin/python3.7
import sys
from lib2to3.main import main

sys.exit(main("lib2to3.fixes"))
